src/Commands/core/CacheCommands.php

```diff
@@ -58,6 +58,23 @@ public function get($cid, $bin = 'default', $options = ['format' => 'json'])
         return new PropertyList($result);
     }
 
+    /**
+     * Invalidate by cache tags.
+     *
+     * @command cache:tags
+     * @param string $tags A comma delimited list of cache tags to clear.
+     * @aliases ct
+     * @bootstrap full
+     * @usage drush cache:tag node-12,user-4
```

Comment:
Would it not be:

`node:12,user:4`

?

Reply:
Also is "cache:tag" the right command? Maybe:

`cache:invalidate-tags` (though that is very long)

Reply:
Fixed tag names. Thanks.

About the command name, I think you are technically correct but I dont think there will be other operations that people want to do with tags besides invalidate. So I'm inclined to keep the briefer name.